samples_project/Assets/SampleViewer/Samples/VRSample/readme.md

```diff
@@ -1,31 +1,41 @@
-2. Drag and Drop the VR Origin Prefab into the scene.
-3. Drag and drop the XR Interaction Manager Prefab into the scene.
-4. Draga an drop the Locomotion System Prefab into the scene.
-5. Parent the XR Origin prefab to the ArcGISMap game object.
-6. Expand the XR Origin fully and select the Main Camera Game Object.
-7. In the Inspector, click add component and search "ArcGIS Camera Component" and add it to the Main Camera Game Object.
-8. Press play and enjoy exploring.
-
-## How to Use
+How to use the sample
 This sample runs off of the OpenXR framework and supports the following headsets/devices:
-1. Oculus Rift S
-2. Oculus Rift
-3. Oculus Quest
-4. Oculus Quest 2
+1. Meta Rift S
+2. Meta Rift
+3. Meta Quest
+4. Meta Quest 2
 5. HTC Vive
 6. HTC Vive Pro
 6. HTC Vive Cosmos
 7. Valve Index
 
-In order to use this sample, connect your headset to your computer and ensure it is detected properly. Upon loading the sample, place the headset on your head. It should automatically use the headset to control the cameras height and rotation as well as position in the world. To move, use the joystick on the right controller, pushing the joystick will move you in that direction. The joystick on the left controller can be used to 'snap turn' 45 degrees, making it so you can turn around quickly.
+## Deployment
+For deploying, please refer to this [document](https://developers.arcgis.com/unity/deployment/)
+In order for the sample to run on Meta Headsets the device, must be placed in **Developer Mode**. Please refer to this [document](https://developer.Meta.com/documentation/native/android/mobile-device-setup/) for how to perform such an action. This must be done for running the sample in the Unity Engine, making a build to run on Windows, or making a build directly to the headset (Meta Quest, Meta Quest 2 and Meta Quest Pro)
+
+1. Ensure your headset is properly connected to your computer. For Meta Headsets, make sure the Meta Desktop App is open and your headset says **connected** in the **devices** section. For Steam VR headsets, ensure Steam is open and Steam VR is running. 
+2. Place the headset on your head. The sample should automatically use the headset to control the camera's position and rotation in the world. 
+3. To move, use the joystick on the left controller, pushing the joystick will move you in that direction. The joystick on the right controller can be used to 'snap turn' 45 degrees, making it so you can turn around quickly. Additionally, pushing the right joystick up or down will allow the player to fly up or down in the world. 
+
+Please ensure that the project is using URP if you are using any of the Meta Headsets listed above. HDRP may be used for any of the other headsets, but not Meta. It will cause errors in the project. 
+
+## Provided Prefabs
```

Comment:
I had a quick chat with Andy and we think it's better to move this information under How it works. It doesn't need to have a specific heading title. So remove the title `## Provided Prefabs` and put the text starting from `This sample provides...` under How it works before the step starts.